v23.1/node-shutdown.md

```diff
@@ -577,6 +579,15 @@ You can use [`cockroach node drain`](cockroach-node.html) to drain a node separa
 <section class="filter-content" markdown="1" data-scope="decommission">
 ### Remove nodes
 
+- [Prerequisites](#prerequisites)
+- [Step 1. Get the IDs of the nodes to decommission](#step-1-get-the-ids-of-the-nodes-to-decommission)
+- [Step 2. Drain the nodes manually](#step-2-drain-the-nodes-manually)
```

Comment:
We've gone back and forth on this a lot.  This is likely an entirely separate discussion from this PR, but at some point it may be worth noting that this step is optional, but can improve the speed of the decommission. It is also highly recommended to drain the node first if the node has any issues (e.g. hardware).